docs/use_cases/multimodal.md

```diff
@@ -39,6 +39,61 @@ maxdepth: 1
 /examples/multi_modal/llava_multi_modal_tesla_10q.ipynb
 ```
 
+### LLaVa-13, Fuyu-8B and MiniGPT-4 Multi-Modal LLM Models Comparison for Image Reasoning
+
+This notebooks shows how to use different Multi-Modal LLM models for image understanding/reasoning. The various model inferences are supported by Replicate. We compared several popular Multi-Modal LLMs:
+
+- GPT4-V (OpenAI API)
+- LLava-13B (Replicate)
+- Fuyu-8B (Replicate)
+- MiniGPT-4 (Replicate)
+
+Check out our guides below:
+
+```{toctree}
+---
+maxdepth: 1
+---
+/examples/multi_modal/replicate_multi_modal.ipynb
+GPT4-V: </examples/multi_modal/openai_multi_modal.ipynb>
+```
+
+### Pydantic Program for Generating Structured Output for Multi-Modal LLMs
+
+This notebooks shows how to generate `structured` output with new OpenAI GPT4V via LlamaIndex. The user just needs to specify a Pydantic object.
+
+Check out our guides below:
```

Comment:
There's only one notebook, so maybe

`You can generate `structured` output with new OpenAI GPT4V via LlamaIndex. The user just needs to specify a Pydantic object.

Check out the guide below: